src/test/resources/hotels/hotels6.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Hotels passwords,emails
+  Scenario: Verify error message for invalid data in SignUp form
+    When I launch hotels website
+    And I click sign in button
+    And I click on SignUp link
+    When I click on email box
+    And I enter this email
+    When I verify error message is displayed
```

Comment:
This is a verification step, should be labeled as Then. Pls correct for all such occurrences.